README.md

```diff
@@ -35,7 +35,7 @@ QCOW variables:
 
 | Name          | Default value                                            |  Description                                                 |
 |---------------|----------------------------------------------------------|--------------------------------------------------------------|
-| miq_qcow_url  | http:////releases.manageiq.org/manageiq-ovirt-fine-1.qc2 | The URL of the ManageIQ QCOW image. |
+| miq_qcow_url  | http://releases.manageiq.org/manageiq-ovirt-fine-1.qc2 | The URL of the ManageIQ QCOW image. |
```

Comment:
Not relevant in this patch, but shouldn't we change the URL in the doc to Gaprindashvili release?
@machacekondra

Reply:
Yep, can you please change the URL to : http://releases.manageiq.org/manageiq-ovirt-gaprindashvili-2.qc2

Reply:
Done.